src/Core/System/SalesChannel/Context/SalesChannelContextFactory.php

```diff
@@ -108,10 +108,14 @@ public function create(string $token, string $salesChannelId, array $options = [
             $itemRounding
         );
 
+        if (\array_key_exists(SalesChannelContextService::DOMAIN_ID, $options) && \is_string($options[SalesChannelContextService::DOMAIN_ID])) {
```

Comment:
we should declare `$domainId = null` before the if condition, so it is clear, why we still need the null coalescing operator